src/components/Battle.js

```diff
@@ -89,48 +100,66 @@ class Battle extends Component {
       moves: state.maxMoves,
       handArray: newHandArray,
       deckArray: newdeckArray,
-      discardArray: newDiscardArray
+      discardArray: newDiscardArray,
+      actionQueue: []
```

Comment:
Forgot to remove this, it's not needed

Reply:
Actually, I don't even know how it ended up there. That whole function need rewriting. :laughing: